src/types/types.ts

```diff
@@ -635,6 +659,16 @@ export enum NftFilters {
   AIRDROPS = 'AIRDROPS'
 }
 
+/**
+ * Enum of ordering that can be applied to a {@link getNftsForOwner} or a
+ * {@link getContractsForOwner} response.
+ *
+ * @beta
+ */
+export enum NftOrdering {
+  TRANSFERTIME = 'TRANSFERTIME'
```

Comment:
nit: rename to `TRANSFER_TIME`

Reply:
This is the name on the backend and I think it's good to have the enum name match the underlying string value so that certain types of string <-> enum conversions are easier